From f50edfa55ae12a6575c502b00f9f42e283cc9c4b Mon Sep 17 00:00:00 2001
From: Timothy Pearson <tpearson@raptorengineeringinc.com>
Date: Mon, 6 Apr 2015 22:02:18 -0500
Subject: [PATCH 11/22] mainboard/lenovo/t400: Increase backlight frequency to
 reduce flicker

TEST: Booted Lenovo T400 with LED backlight and verified the absence
of any visible flicker along with proper operation of the brightness
control.

Change-Id: Ifb485f4aad385aac794978527fd7d246f444ea08
Signed-off-by: Timothy Pearson <tpearson@raptorengineeringinc.com>
---
 src/mainboard/lenovo/t400/devicetree.cb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/mainboard/lenovo/t400/devicetree.cb b/src/mainboard/lenovo/t400/devicetree.cb
index cc27d25..6280b91 100644
--- a/src/mainboard/lenovo/t400/devicetree.cb
+++ b/src/mainboard/lenovo/t400/devicetree.cb
@@ -5,6 +5,8 @@ chip northbridge/intel/gm45
 	register "gfx.link_frequency_270_mhz" = "1"
 	register "gfx.lvds_num_lanes" = "4"
 
+	register "gfx.backlight" = "0x620062"
+
 	device cpu_cluster 0 on
 		chip cpu/intel/socket_BGA956
 			device lapic 0 on end
-- 
1.9.1

